### PR TITLE
[Frontend] Add a note about stimulusFetch & TypeScript config

### PR DESCRIPTION
**Pull request by jmsche**
Just a note about using `stimulusFetch` in Stimulus controllers while writing them with TypeScript.

### CODE REVIEW

File: frontend/encore/simple-example.rst [hunk 1]
```diff
@@ -317,6 +317,11 @@ split to *separate* files by Encore. Then, those files won't be downloaded until
 the moment a matching element (e.g. ``<div data-controller="lazy-example">``)
 appears on the page!
 
+.. note::
+
+    If you write your controllers using TypeScript, make sure
+    ``removeComments`` is not set to ``true`` in your TypeScript config.
```
Comment (OskarStark) [inline frontend/encore/simple-example.rst:323]: Maybe extend the sentence to explain the "why"?

Comment (jmsche) [reply frontend/encore/simple-example.rst:323]: Not sure if this is necessary? I mean, the config is a comment, so it makes sense without explaining it? Or do you mean to explain that TypeScript is compiled before stimulusFetch config is handled by Simulus bridge?

Comment (weaverryan) [reply frontend/encore/simple-example.rst:323]: I like the note as is :)

**Review by weaverryan — APPROVED** [review]

**Review by maxhelias — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you @jmsche.

**Pull request merged**